manifest.json

```diff
@@ -13,11 +13,14 @@
     "webRequestBlocking",
     "*://*.ft.com/",
     "*://*.herokuapp.com/",
+		"*://next-elastic.ft.com/*",
+		"*://search-next-elasticsearch-[a-z1-9]+.(eu-west-1|us-east-1).es.amazonaws.com/*",
```

Comment:
Manifests do not support regex for url matches, instead they have their own limit pattern matching https://developer.chrome.com/extensions/match_patterns

I think we will have to support this in the same way that we support `herokuapp.com` domains with the additional check in background.js https://github.com/GlynnPhillips/vary-headers/pull/6/files#diff-f9b12bd72f40c9b8ed144484f0f12a5fR47https://github.com/GlynnPhillips/vary-headers/pull/6/files#diff-f9b12bd72f40c9b8ed144484f0f12a5fR47